data/fields/crossing/signals.json

```
@@ -0,0 +1,8 @@
+{
+    "key": "crossing:signals",
+    "type": "defaultCheck",
```

```suggestion
    "type": "check",
```

See https://github.com/ideditor/schema-builder?tab=readme-ov-file#checkboxes